miio/tests/test_philips_bulb.py

```diff
@@ -0,0 +1,99 @@
+from unittest import TestCase
+from miio import PhilipsBulb
+from .dummies import DummyDevice
+import pytest
+
+
+class DummyPhilipsBulb(DummyDevice, PhilipsBulb):
+    def __init__(self, *args, **kwargs):
+        self.state = {
+            'power': 'on',
+            'bright': 85,
+            'cct': 9,
+            'snm': 0,
+            'dv': 0
+        }
+        self.return_values = {
+            'get_prop': self._get_state,
+            'set_power': lambda x: self._set_state("power", x),
+        }
+        super().__init__(args, kwargs)
+
+
+@pytest.fixture(scope="class")
+def philips_bulb(request):
+    request.cls.device = DummyPhilipsBulb()
+    # TODO add ability to test on a real device
+
+
+@pytest.mark.usefixtures("philips_bulb")
+class TestPhilipsBulb(TestCase):
+    def is_on(self):
+        return self.device.status().is_on
+
+    def state(self):
+        return self.device.status()
+
+    def test_on(self):
+        self.device.off()  # ensure off
+
+        start_state = self.is_on()
+        assert start_state is False
+
+        self.device.on()
+        assert self.is_on() is True
+
+    def test_off(self):
+        self.device.on()  # ensure on
+
+        assert self.is_on() is True
+        self.device.off()
+        assert self.is_on() is False
+
+    def test_status(self):
+        self.device._reset_state()
+
+        assert self.is_on() is True
+        assert self.state().brightness == self.device.start_state["bright"]
+        assert self.state().color_temperature == self.device.start_state["cct"]
+        assert self.state().scene == self.device.start_state["snm"]
+        assert self.state().delay_off_countdown == self.device.start_state["dv"]
+
+    def set_brightness(self):
+        def brightness():
+            return self.device.status().brightness
+
+        self.device.set_brightness(10)
+        assert brightness() == 10
+        self.device.set_brightness(20)
+        assert brightness() == 20
```

Comment:
Above comment on incorrect values. Merging with "brightness adjustable" could be done at some point?